src/shortcuts/__tests__/redo.ts

```diff
@@ -49,20 +49,23 @@ it('group contiguous navigation actions preceding a thought change on redo', ()
   const store = createTestStore()
 
   store.dispatch([importText(RANKED_ROOT, `
-      - a
-      - b`
+    - a
+    - b`
   ),
   { type: 'cursorDown' },
+  { type: 'setCursor', thoughtsRanked: [{ value: 'b', rank: 1 }] },
```

Comment:
I think the `cursor` is already on `b` after the `cursorDown`.

Reply:
No, the cursor is on `a`

Reply:
Ah, you're right. Thanks!